packages/core/src/api/serialization/html/internalHTMLSerializer.ts

```diff
@@ -0,0 +1,148 @@
+// TODO: IMO this should be part of the formatConversions file since the custom
```

Comment:
not sure about this? I think separating is nice?

Reply:
Ah yeah this is an old comment, will remove.